docs/dev/dev_environment.md

```diff
@@ -0,0 +1,333 @@
+# Decide On Development Environment
```

Comment:
File needs to be updated to remove cookie cutter and ensure it works as expected

Reply:
Replaced cookiecutter templating.